{
  "comments": [
    {
      "key": {
        "uuid": "d27b40d5_c119322c",
        "filename": "src/PSUSensor.cpp",
        "patchSetId": 9
      },
      "lineNbr": 129,
      "author": {
        "id": 1000418
      },
      "writtenOn": "2021-06-07T18:01:33Z",
      "side": 1,
      "message": "Looks like it is global errno. \nCan you tell me which file include brings it here ? I am not able to find.",
      "range": {
        "startLine": 129,
        "startChar": 12,
        "endLine": 129,
        "endChar": 18
      },
      "revId": "2f561de79f3f3616c642e2f5b2a924ea3b5d3db6",
      "serverId": "adbd0a64-1f21-4d83-b585-671fe73cb6e4",
      "unresolved": true
    },
    {
      "key": {
        "uuid": "2eb4c329_7efe1dc3",
        "filename": "src/PSUSensor.cpp",
        "patchSetId": 9
      },
      "lineNbr": 130,
      "author": {
        "id": 1000418
      },
      "writtenOn": "2021-06-07T18:01:33Z",
      "side": 1,
      "message": "It can throw exception, right ? Does it need to be under try-catch block ?\nhttps://en.cppreference.com/w/cpp/string/basic_string/stof",
      "range": {
        "startLine": 130,
        "startChar": 23,
        "endLine": 130,
        "endChar": 39
      },
      "revId": "2f561de79f3f3616c642e2f5b2a924ea3b5d3db6",
      "serverId": "adbd0a64-1f21-4d83-b585-671fe73cb6e4",
      "unresolved": true
    },
    {
      "key": {
        "uuid": "6304d361_7acb996d",
        "filename": "src/PSUSensor.cpp",
        "patchSetId": 9
      },
      "lineNbr": 145,
      "author": {
        "id": 1000418
      },
      "writtenOn": "2021-06-07T18:01:33Z",
      "side": 1,
      "message": "Not required. RAII takes care.",
      "range": {
        "startLine": 145,
        "startChar": 7,
        "endLine": 145,
        "endChar": 23
      },
      "revId": "2f561de79f3f3616c642e2f5b2a924ea3b5d3db6",
      "serverId": "adbd0a64-1f21-4d83-b585-671fe73cb6e4",
      "unresolved": true
    }
  ]
}